clang/unittests/AST/ASTImporterTest.cpp

```diff
@@ -9284,6 +9284,26 @@ TEST_P(ASTImporterOptionSpecificTestBase,
   // EXPECT_EQ(ToF1Imported->getPreviousDecl(), ToF1);
 }
 
+TEST_P(ASTImporterOptionSpecificTestBase, ImportFunctionAutoType) {
```

Comment:
This test belongs to `ImportAutoFunctions` and a better name is `ReturnWithSubstNonTypeTemplateParmExpr`.

Reply:
Code has been fixed.